pkg/rpcclient/nns/contract.go

```diff
@@ -44,6 +71,21 @@ func NewReader(invoker Invoker, hash util.Uint160) *ContractReader {
 	return &ContractReader{*nep11.NewNonDivisibleReader(invoker, hash), invoker, hash}
 }
 
+// Hash stores the hash of the native NEOToken contract.
+var Hash = state.CreateNativeContractHash(nativenames.Neo)
```

Comment:
Please, be more careful. It's NNS.

Reply:
BTW, we need a dynamic hash for this wrapper. Have you tried to run `generate-rpcwrapper` command for NNS manifest deployed on mainnet? Run it without `--hash` flag. Take a look at the output. It may be taken as base.